From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Tue, 8 Mar 2022 12:11:34 +0100
Subject: [PATCH] clk: imx6: fix cko1_sels

cko1 outputs pll5_video_div, not pll5_video. Also it outputs
pll4_audio_div, not pll4_audio.

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
---
 drivers/clk/imx/clk-imx6.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/imx/clk-imx6.c b/drivers/clk/imx/clk-imx6.c
index 56ae09fc7bdc..4f843b8ba735 100644
--- a/drivers/clk/imx/clk-imx6.c
+++ b/drivers/clk/imx/clk-imx6.c
@@ -166,7 +166,7 @@ static const char *cko1_sels[] = {
 	"pll3_usb_otg",
 	"pll2_bus",
 	"pll1_sys",
-	"pll5_video",
+	"pll5_video_div",
 	"video_27m",
 	"axi",
 	"enfc",
@@ -178,7 +178,7 @@ static const char *cko1_sels[] = {
 	"ipg",
 	"ipg_per",
 	"ckil",
-	"pll4_audio",
+	"pll4_audio_div",
 };
 
 static const char *cko2_sels[] = {
